sphinx/source/docs/releases/0.12.4.rst

```diff
@@ -125,6 +125,13 @@ BokehJS has migrated to using ``.tsx`` templates. Provisional support for
 including ``.eco`` templates has been removed from both BokehJS and the
 Bokeh Python library.
 
+Python 3.3 No Longer Supported
```

Comment:
The b64encode change was made in commit 8e316faf2a12975c299bdde5fb002c2435f92642 29/Dec/2016; the 0.12.3 tag predates that (7/Oct/2016).